src/utils/recompiler.ts

```diff
@@ -0,0 +1,127 @@
+import fs from "fs";
+import path from "path";
+import { createHash } from "crypto";
+import { HardhatRuntimeEnvironment, TaskArguments } from "hardhat/types";
+import { starknetCompileAction } from "../task-actions";
+
+const fsPromises = fs.promises;
+
+// FileName - HashPair
+interface NameHashPair {
+    [key: string]: string;
+}
+// Cache file name
+export const CACHE_FILE_NAME = "cairo-files-cache.json";
```

Comment:
If this is not used outside of the file, it doesn't have to be exported.

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/106/commits/d37016745892a386d20948318b9b400389379d34